src/environmentd/src/http/sql.rs

```diff
@@ -830,3 +866,69 @@ fn is_txn_exit_stmt(stmt: &Statement<Raw>) -> bool {
         Statement::Commit(_) | Statement::Rollback(_) | Statement::Prepare(_)
     )
 }
+
+#[cfg(test)]
+mod tests {
+    use super::WebSocketAuth;
+
+    #[test]
+    fn smoke_test_websocket_auth_parse() {
+        let basic = "{ \"user\": \"mz\", \"password\": \"1234\" }";
```

Comment:
Light preference to use `r#` syntax here for easier JSON reading and writing.

Reply:
Good idea, that definitely makes it cleaner!